packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -161,14 +165,18 @@ const SelectArrayInput: FunctionComponent<
         ...rest
     } = props;
     const classes = useStyles(props);
+    const inputLabel = useRef(null);
+    const [labelWidth, setLabelWidth] = useState(0);
+    useEffect(() => {
+        setLabelWidth(inputLabel.current.offsetWidth);
```

Comment:
inputLabel.current can be null at mounting. Can you add a check?

Reply:
I don't think so. the effect will run after the first render, and at that time, the ref will be defined.